Buildings/Controls/OBC/ASHRAE/G36/AHUs/MultiZone/VAV/Validation/Controller.mo

```diff
@@ -115,11 +115,20 @@ model Controller "Validation controller model"
     final period=3600)
     "Freeze protection reset"
     annotation (Placement(transformation(extent={{-240,-170},{-220,-150}})));
-  Buildings.Controls.OBC.CDL.Logical.TrueFalseHold truFalHol(
-    final trueHoldDuration=1)
-    "Break loop"
-    annotation (Placement(transformation(extent={{220,30},{240,50}})));
-
+  Buildings.Controls.OBC.CDL.Conversions.BooleanToReal booToRea(
+    final realTrue=1,
+    final realFalse=0)
+    "Convert to real"
+    annotation (Placement(transformation(extent={{190,70},{170,90}})));
+  Buildings.Fluid.BaseClasses.ActuatorFilter fil(
```

Comment:
We should not use block that is not in the `OBC` package.

Reply:
Addressed with afb1c60